### PR TITLE
Pages plugin

### LINKED ISSUE #102: Footer information
As an internet user, I expect there to be a footer containing links to Legal information, data protection information and a link to the software used.

We could go the route of creating a Markdown-mini-crm-Pages model that can be enabled to be visible publicly and in the footer.
Or we hardcode something.

### PR DESCRIPTION
**Pull request by felixrindt**
Fix #102

### CODE REVIEW

**Comment by felixrindt** [comment]
image

-------------

image

File: ephios/templates/base.html [hunk 1]
```diff
@@ -102,6 +102,14 @@
             {% block content %}{% endblock %}
         </div>
     </div>
+    <footer class="footer text-center mt-auto py-3">
+        <small>
+            {% for label, url in footer.items %}
+                <a href="{{ url }}">{{ label }}</a> ·
+            {% endfor %}
+            <span class="text-muted">powered by <a href="https://ephios.de/" target="_blank">ephios</a></span>
```
Comment (jeriox) [inline ephios/templates/base.html:110]: translation?

File: ephios/templates/base.html [hunk 2]
```diff
@@ -102,6 +102,14 @@
             {% block content %}{% endblock %}
         </div>
     </div>
+    <footer class="footer text-center mt-auto py-3">
```
Comment (jeriox) [inline ephios/templates/base.html:105]: i'd prefer to have a footer that sticks to the bottom of the viewport (like so: https://getbootstrap.com/docs/4.1/examples/sticky-footer/)

File: ephios/plugins/pages/views.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+from django.views.generic import DetailView
+
+from ephios.plugins.pages.models import Page
+
+from django.core.exceptions import PermissionDenied
+
+
+class PageView(DetailView):
+    model = Page
+    accept_global_perms = True
```
Comment (jeriox) [inline ephios/plugins/pages/views.py:10]: do we need this?

**Review by jeriox — COMMENTED** [review]
works like a charm!

File: ephios/templates/base.html (hunk 2 above)
Comment (jeriox) [reply ephios/templates/base.html:105]: (but with your design, their big grey bar is ugly)

Comment (felixrindt) [reply ephios/templates/base.html:105]: I'll try :)

File: ephios/plugins/pages/views.py (hunk 3 above)
Comment (felixrindt) [reply ephios/plugins/pages/views.py:10]: nope

**Review by jeriox — APPROVED** [review]
looks better, but the viewport stuff is still a bit buggy in chrome on android

**Pull request merged**